app/navigation/Navigator.scala

```diff
@@ -52,6 +52,23 @@ class RootNavigator @Inject()() extends Navigator {
   override def defaultNormalMode: Call = controllers.routes.IndexController.onPageLoad()
 
   override def defaultCheckMode: Call = controllers.routes.IndexController.onPageLoad()
+
+  private def isBasicDetailsIncomplete(srn: Srn, userAnswers: UserAnswers): Boolean = {
```

Comment:
Can we use existing `TaskListStatusUtils.getBasicDetailsTaskListStatus` for the same?